### PR TITLE
Add description for TLS related flags for redis

### PR DESCRIPTION
**Pull request by Lagovas**
Updated tools with new flags

### CODE REVIEW

**Review by vixentael — COMMENTED** [review]
do i understand correctly that these are the same flags, copy-pasted in 16 files?

File: content/acra/configuring-maintaining/general-configuration/acra-addzone.md [hunk 1]
```diff
@@ -60,6 +60,134 @@ weight: 10
...
+  * `1` — request client certificate, but don't require it;
+  * `2` — expect to receive at least one certificate to continue the handshake;
+  * `3` — don't require client certificate, but validate it if client actually sent it;
+  * `4` — (default) request and validate client certificate.
+
+  These values correspond to [crypto.tls.ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType).
+
+* `--redis_tls_client_ca=<filename>`
+
+  Path to additional CA certificate for Redis' certificate validation (overrides `--tls_ca`).
```
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-addzone.md:78]: > overrides `--tls_ca`

do we have this flag in acra-addzone?

File: content/acra/configuring-maintaining/general-configuration/acra-addzone.md [hunk 2]
```diff
@@ -60,6 +60,134 @@ weight: 10
...
+* `--redis_tls_crl_client_from_cert=<policy>`
+
+  How to treat CRL's URL described in a certificate from Redis server
+
+  * `use` — try URL(s) from certificate after the one from configuration (if set)
+  * `trust` — try first URL from certificate, if it does not contain checked certificate, stop further checks
+  * `prefer` — (default) try URL(s) from certificate before the one from configuration (if set)
+  * `ignore` — completely ignore CRL's URL(s) specified in certificate
+
+  "URL from configuration" above means the one configured with `--tls_crl_*_url` flags.
```
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-addzone.md:134]: `--tls_crl_*_url` — where this is coming from? should we put a link here? i haven't found any `tls_crl_...` flags for acra-addzone

**Comment by Lagovas** [comment]
> do i understand correctly that these are the same flags, copy-pasted in 16 files?
actually yes, but it was wrong and you catched.

> do we have this flag in acra-addzone?
no, only in acra-server/acra-translator services. so removed from all other

> --tls_crl_*_url — where this is coming from? should we put a link here? i haven't found any tls_crl_... flags for acra-addzone

same here, clarified flag that means in the note for all services except AS

File: content/acra/configuring-maintaining/general-configuration/acra-server/_index.md [hunk 3]
```diff
@@ -76,6 +76,133 @@ weight: 3
...
+* `--redis_tls_crl_client_from_cert=<policy>`
+
+  How to treat CRL's URL described in a certificate from Redis server
+
+  * `use` — try URL(s) from certificate after the one from configuration (if set)
+  * `trust` — try first URL from certificate, if it does not contain checked certificate, stop further checks
+  * `prefer` — (default) try URL(s) from certificate before the one from configuration (if set)
+  * `ignore` — completely ignore CRL's URL(s) specified in certificate
+
+  "URL from configuration" above means the one configured with `--tls_crl_*_url` flags.
```
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:150]:
```suggestion
  "URL from configuration" above means the one configured with `--tls_crl_*_url` flags. See [Configuring & maintaining > TLS > CRL](/acra/configuring-maintaining/tls/crl/).
```

File: content/acra/configuring-maintaining/general-configuration/acra-server/_index.md [hunk 4]
```diff
@@ -76,6 +76,133 @@ weight: 3
...
+* `--redis_tls_ocsp_client_from_cert=<policy>`
+
+  How to treat OCSP server URL described in a certificate from Redis server
+
+  * `use` — try URL(s) from certificate after the one from configuration (if set)
+  * `trust` — try URL(s) from certificate, if server returns "Valid", stop further checks
+  * `prefer` — (default) try URL(s) from certificate before the one from configuration (if set)
+  * `ignore` — completely ignore OCSP's URL(s) specified in certificate
+
+  "URL from configuration" above means the one configured with `--tls_ocsp_*_url` flags.
```
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:188]:
```suggestion
  "URL from configuration" above means the one configured with `--tls_ocsp_*_url` flags, see [Configuring & maintaining > TLS > OCSP](/acra/configuring-maintaining/tls/ocsp/).
```

**Review by vixentael — CHANGES_REQUESTED** [review]
Thank you! I have a small suggestion, and no further comments

**Review by vixentael — APPROVED** [review]
Great! Please don't forget to merge into stable

**Pull request merged**